README.md

```diff
@@ -72,6 +72,35 @@ const element = ReactSelector('componentName').withProps({
 });
 ```
 
+#### Searching for nested components
+
+You can search for a desired element among the component's children using the `findReact(element)` method. The method takes the element name or tag name as a parameter.
+
+```js
+import ReactSelector from 'testcafe-react-selectors';
+
+const TodoApp          = ReactSelector('TodoApp');
+const TodoList         = ReactSelector('TodoList');
+const itemsCountStatus = TodoList.findReact('div');
+const itemsCount       = TodoList.findReact('div span');
+```
+
+You can call the `findReact` method in a chain, for example:
+
+```js
+import ReactSelector from 'testcafe-react-selectors';
+
+const itemsCount = ReactSelector('TodoApp').findReact('TodoList').findReact('span');
+```
+
+You can also combine `.findReact` with regular selectors and [other](](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#functional-style-selectors)) methods like `.find` or `.withText`, for example:
+
+```js
+import ReactSelector from 'testcafe-react-selectors';
+
+var itemsCount = ReactSelector('TodoApp').find('.items-count').findReact('span');
+```
+
```

Comment:
From what is said in this topic, I don't understand the difference between `findReact` and `find` used for `ReactSelector`. Is it that `findReact` can search for components in addition to elements?

Reply:
What's going to happen if we write 
```
var itemsCount = ReactSelector('TodoApp').find('.items-count').find('span');
```

Reply:
In this case you will look for `.items-count span` elements inside of `TodoApp` component